paper-pnas.tex

```diff
@@ -0,0 +1,220 @@
+
+\title{How and Why to run a Hack Week}
+
+% Use letters for affiliations, numbers to show equal authorship (if applicable) and to indicate the corresponding author
+\author[a,b,1]{Daniela Huppenkothen}
+\author[c,1,2]{Anthony Arendt} 
+\author[b,d]{David W. Hogg}
+\author[e]{Karthik Ram}
+\author[f]{Jake VanderPlas}
+\author[g]{Ariel Rokem}
+
+\affil[a]{Center for Data Science, New York University, 65 5th Avenue, 7th Floor, New York, NY 10003, USA}
+\affil[b]{Center for Cosmology and Particle Physics, New York University, 726 Broadway, 10th Floor, New York, NY 10003, USA}
+\affil[c]{}
+\affil[d]{Center for Computational Astrophysics, Flatiron Institute, 162 5th Ave, New York, NY 10010, USA}
+\affil[e]{}
+\affil[f]{}
+\affil[g]{The University of Washington eScience Institute, The WRF Data Science Studio, Physics/Astronomy Tower, 6th Floor, 3910 15th Ave NE, Campus Box 351570, University of Washington, Seattle, WA 98105, USA}
+
+% Please give the surname of the lead author for the running footer
+\leadauthor{Huppenkothen} 
+
+% Please add here a significance statement to explain the relevance of your work
+\significancestatement{As scientific disciplines grapple with more data sets of rapidly increasing complexity and size, new approaches are urgently required to introduce new statistical and computational tools into research communities and improve the cross-disciplinary exchange of ideas. In this paper, we introduce a new type of scientific workshop, called a \textit{hack week}, which allows for fast dissemination of new methodologies into scientific communities, and fosters exchange and collaboration within and between disciplines. We present implementations of this concept in astronomy, neuroscience and geoscience, and show that hack weeks produce positive learning outcomes, foster lasting collaborations, yield scientific results and improve attitudes toward open science and reproducibility.}
+
+% Please include corresponding author, author contribution and author declaration information
+\authorcontributions{The concept was designed by Jake VanderPlas, the paper design initially by David W. Hogg, Karthik Ram and Jake VanderPlas. Daniela Huppenkothen, Ariel Rokem and Anthony Arendt performed the studies and the data analysis. All authors contributed to the final manuscript.}
+
+\authordeclaration{The authors declare no conflicts of interest.}
+%\equalauthors{\textsuperscript{1}A.O.(Author One) and A.T. (Author Two) contributed equally to this work (remove if not applicable).}
+\correspondingauthor{\textsuperscript{1}To whom correspondence should be addressed. E-mail: daniela.huppenkothen@nyu.edu}
+
+% Keywords are not mandatory, but authors are strongly encouraged to provide them. If provided, please include two to five keywords, separated by the pipe symbol, e.g:
+\keywords{Data science $|$ Education $|$ Reproducibility $|$ Interdisciplinary Collaboration $I$ Astronomy $|$ Neurosciences $|$ Geosciences} 
```

Comment:
"Neurosciences" should be "Neuroscience"

Also: the delimiter between "Interdisciplinary Collaboration" and "Astronomy" should be a vertical line.